We used the recommended settings from (Ding et al. 2015) to map the sequencing reads and compute reactivities of the annotated transcripts using the Galaxy tools (Afgan et al. 2018) Bowtie-2 and StructureFold (Langmead and Salzberg 2012; Tang et al. 2015). We selected the U1 homolog transcript bearing the largest secondary structure distance between the unconstrained and SHAPE-constrained structure prediction (using RNAfold).


- Afgan, Enis, Dannon Baker, Bérénice Batut, Marius van den Beek, Dave Bouvier, Martin Čech, John Chilton, et al. 2018. “The Galaxy Platform for Accessible, Reproducible and Collaborative Biomedical Analyses: 2018 Update.” Nucleic Acids Research, gky379. https://doi.org/10.1093/nar/gky379.

- Ding, Yiliang, Chun Kit Kwok, Yin Tang, Philip C Bevilacqua, and Sarah M Assmann. 2015. “Genome-Wide Profiling of in Vivo Rna Structure at Single-Nucleotide Resolution Using Structure-Seq.” Nature Protocols 10 (7). Nature Publishing Group: 1050.

- Langmead, Ben, and Steven L Salzberg. 2012. “Fast Gapped-Read Alignment with Bowtie 2.” Nature Methods 9 (4). Nature Publishing Group: 357.

- Tang, Yin, Emil Bouvier, Chun Kit Kwok, Yiliang Ding, Anton Nekrutenko, Philip C Bevilacqua, and Sarah M Assmann. 2015. “StructureFold: Genome-Wide Rna Secondary Structure Mapping and Reconstruction in Vivo.” Bioinformatics 31 (16). Oxford University Press: 2668–75.
